export-ingester/export_ingester/ingest.py

```diff
@@ -38,40 +38,39 @@ class ExportIngester:
     def __init__(
         self,
         api_client: APIClient,
-        sftp_client: SFTPClient,
+        ssh_client: SSHClient,
     ):
         self.api_client = api_client
-        self.sftp_client = sftp_client
+        self.ssh_client = ssh_client
 
     async def ingest(
         self,
         remote_path: str,
         params: ExportParams,
     ):
-        await self.sftp_client.pipe_exports(
+        await self.ssh_client.pipe_exports(
             remote_path,
             await self.api_client.get_exports(params),
         )
 
+    async def run_sbatch_command(self, sbatch_command, remote):
+        """Runs the sbatch command on the remote server via SSH."""
+        await self.ssh_client.remote_sbatch(sbatch_command, remote)
+
 
 @asynccontextmanager
 async def get_managed_export_ingester(
     settings: Settings,
 ) -> AsyncGenerator[ExportIngester, None]:
     stack = await AsyncExitStack().__aenter__()
 
-    sftp_client_factory = SFTPClientFactory(
+    ssh_client_factory = SSHClientFactory(
         settings.SFTP_USERNAME,
         settings.SFTP_PASSWORD,
         settings.SFTP_HOST,
     )
-
     http_client = await stack.enter_async_context(httpx.AsyncClient())
-    sftp_client = await stack.enter_async_context(sftp_client_factory.get_sftp_client())
-
-    yield ExportIngester(
-        APIClient(str(settings.BASE_URL), http_client),
-        sftp_client,
-    )
+    ssh_client = await stack.enter_async_context(ssh_client_factory.get_ssh_client())
+    yield ExportIngester(APIClient(str(settings.BASE_URL), http_client), ssh_client)
```

Comment:
Can you dd a trailing `,` after `http_client`? Ruff will then reformat to make this go over multiple lines